src/app/services/technical-record/technical-record.service.ts

```diff
@@ -143,9 +143,10 @@ export class TechnicalRecordService {
       select(selectRouteNestedParams),
       map(params => {
         const createdAt = params['techCreatedAt'];
-        const isProvisional = this.router.url.split('/').pop() === 'provisional';
+        const lastTwoUrlParts = this.router.url.split('/').slice(-2)
```

Comment:
This doesn't seem like an exact science. Can this break if the route paths change?

Reply:
I'm not sure, in what way do you mean break? As in it would become undefined?